doc/apiary/v2/fiware-ngsi-simple.apib

```diff
@@ -89,54 +203,6 @@ Detailed description
          }
         ]
 
-
-### Entity Pagination [GET /v2/entities{?limit,offset,count}]
```

Comment:
As discussed by skype, removing fiware-ngsiv2-examples (it will be added in a next PR). Fixed in 9967613